src/System.Private.CoreLib/shared/Interop/Unix/System.Native/Interop.GetHostName.cs

```diff
@@ -35,7 +35,7 @@ internal static unsafe string GetHostName()
             // If the hostname is truncated, it is unspecified whether the returned buffer includes a terminating null byte.
             name[ArrLength - 1] = 0;
 
-            return Marshal.PtrToStringAnsi((IntPtr)name);
+            return Marshal.PtrToStringAnsi((IntPtr)name)!;
```

Comment:
`Marshal.PtrToStringAnsi` only returns `null` if the parameter is `IntPtr.Zero` or a Win32 atom, so this shouldn't be `null` here based on what `name` can be (same goes for `GetUnixName` and `GetAllMountPoints` below).